### ISSUE TITLE
[HELP] Error download specific formats

### ISSUE THREAD

**Issue by pinguim16** [yt-dlp/yt-dlp#6948]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a new site support request
- [X] I've verified that I'm running yt-dlp version **2023.03.04** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that none of provided URLs [violate any copyrights](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy) or contain any [DRM](https://en.wikipedia.org/wiki/Digital_rights_management) to the best of my knowledge
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and am willing to share it if required

### Region

Brasil

### Example URLs

https://www.youtube.com/watch?v=4uh_MGGXP8I

### Provide a description that is worded well enough to be understood

How can I force download youtube videos that are having problems downloading in a specific format? Does yt-dlp try another format?
Because I have several .part files that do not continue the download.
Command:
start cmd.exe /c ..\yt-dlp_min.exe -P %location% --cookies %location%\00cookie.txt --download-archive %location%\00archive.txt %url% --ignore-errors -4 --force-write-archive

![image](https://user-images.githubusercontent.com/8548495/235312433-c32aef3b-b132-40a8-b6ee-9ab203ed7684.png)

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Adding to archive: youtube Oe0zX-bIf5M
[download] Downloading item 360 of 2089
[youtube] Extracting URL: https://www.youtube.com/watch?v=BmN_UVA9Xr0
[youtube] BmN_UVA9Xr0: Downloading webpage
[youtube] BmN_UVA9Xr0: Downloading android player API JSON
[debug] Loading youtube-nsig.0c487f05 from cache
[debug] [youtube] Decrypted nsig Y_roIgI9_4-tIyb1N => -a0dnQT-vbnl-w
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[debug] Default format spec: best/bestvideo+bestaudio
[info] BmN_UVA9Xr0: Downloading 1 format(s): 22
[debug] Invoking http downloader on "https://rr5---sn-b8u-jo4e.googlevideo.com/videoplayback?expire=1682807068&ei=vERNZO3UOsSj1sQPksEj&ip=187.58.217.237&id=o-ABpMlz_Ek-xvNzHV6lXEG1BzTW53WbwJ7ahzpWIKM_Dx&itag=22&source=youtube&requiressl=yes&mh=f5&mm=31%2C29&mn=sn-b8u-jo4e%2Csn-gpv7kn7d&ms=au%2Crdu&mv=m&mvi=5&pcm2cms=yes&pl=23&pcm2=yes&initcwndbps=1357500&spc=qEK7B3NP4Vta86PwNiOArqXYnlxRbuo&vprv=1&svpuc=1&mime=video%2Fmp4&cnr=14&ratebypass=yes&dur=4902.475&lmt=1674771155925434&mt=1682785018&fvip=5&fexp=24007246&c=ANDROID&txp=7218224&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cpcm2%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Ccnr%2Cratebypass%2Cdur%2Clmt&sig=AOq0QJ8wRAIgBVQG4xto3EFqBPepHlKUZ1R8WUNQ2OLOYZep_DgGPPMCIEPbsKQfHTSTu_Ps-fKmgBWnN8dfRpcYkyXXsM6EkWBD&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpcm2cms%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRAIgOH-bVWk2ctKyzmUPykyFHOsKQKx3W_BT8KWpSGzOL_QCID6ItrfRoJpqah0aRItLs1yISgFt5ZmvT5wT8UmZHC9J"
[debug] File locking is not supported. Proceeding without locking
[download] Destination: Adaptação e solução [BmN_UVA9Xr0].mp4
[download]   0.6% of  373.28MiB at   12.48MiB/s ETA 00:29[download] Got error: Downloaded 2425056 bytes, expected 391415011 bytes. Retrying (1/10)...


ERROR: Did not get any data blocks
  File "__main__.py", line 17, in <module>
  File "yt_dlp\__init__.pyc", line 967, in main
  File "yt_dlp\__init__.pyc", line 957, in _real_main
  File "yt_dlp\YoutubeDL.pyc", line 3369, in download
  File "yt_dlp\YoutubeDL.pyc", line 3344, in wrapper
  File "yt_dlp\YoutubeDL.pyc", line 1507, in extract_info
  File "yt_dlp\YoutubeDL.pyc", line 1518, in wrapper
  File "yt_dlp\YoutubeDL.pyc", line 1615, in __extract_info
  File "yt_dlp\YoutubeDL.pyc", line 1744, in process_ie_result
  File "yt_dlp\YoutubeDL.pyc", line 1888, in __process_playlist
  File "yt_dlp\YoutubeDL.pyc", line 1518, in wrapper
  File "yt_dlp\YoutubeDL.pyc", line 1920, in __process_iterable_entry
  File "yt_dlp\YoutubeDL.pyc", line 1744, in process_ie_result
  File "yt_dlp\YoutubeDL.pyc", line 1888, in __process_playlist
  File "yt_dlp\YoutubeDL.pyc", line 1518, in wrapper
  File "yt_dlp\YoutubeDL.pyc", line 1920, in __process_iterable_entry
  File "yt_dlp\YoutubeDL.pyc", line 1694, in process_ie_result
  File "yt_dlp\YoutubeDL.pyc", line 1507, in extract_info
  File "yt_dlp\YoutubeDL.pyc", line 1518, in wrapper
  File "yt_dlp\YoutubeDL.pyc", line 1615, in __extract_info
  File "yt_dlp\YoutubeDL.pyc", line 1674, in process_ie_result
  File "yt_dlp\YoutubeDL.pyc", line 2779, in process_video_result
  File "yt_dlp\YoutubeDL.pyc", line 3247, in process_info
  File "yt_dlp\YoutubeDL.pyc", line 2970, in dl
  File "yt_dlp\downloader\common.pyc", line 444, in download
  File "yt_dlp\downloader\http.pyc", line 377, in real_download
  File "yt_dlp\downloader\http.pyc", line 348, in download
  File "yt_dlp\YoutubeDL.pyc", line 1015, in report_error
  File "yt_dlp\YoutubeDL.pyc", line 944, in trouble
```

**Comment by pukkandan** [comment]
See pinned issue about format 22

**Comment by pukkandan** [comment]
tldr; Install ffmpeg

**Comment by pinguim16** [comment]
locationff=ffmpeg-master-latest-win64-gpl
-f 22 --ffmpeg-location %locationff% ... correct?

**Comment by pukkandan** [comment]
Remove -f22
